src/core/reducers/addons.js

```diff
@@ -346,6 +371,24 @@ export default function addonsReducer(
         bySlug,
       };
     }
+    case UNLOAD_ADDON: {
```

Comment:
Why not adjust `UnloadAddonReviewsAction` to have `addon` in the payload and have this switch case look for `UNLOAD_ADDON_REVIEWS`? In other words, I don't think you need a separate `unloadAddon()` action. I can't imagine any situation other than deleting reviews where we would need a dedicated `unloadAddon()` action.